### DISCUSSION TITLE
Press enter to select the first matching tag, category tool in recipe editor

### DISCUSSION THREAD

**Discussion by simgunz** [mealie-recipes/mealie#3360] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

In the recipe editor when adding a tag/category/tool I need to use the arrow down to select the first entry matching the query I typed before I can select it with Enter.

Related: https://github.com/mealie-recipes/mealie/issues/3359

### Please provide a concise description of the feature that would resolve your issue.

In the recipe editor when adding a tag/category/tool I would like to be able to press enter to select the first matching entry (as it already happen for the ingredient field for example).

### Please consider and list out some caveats or tradeoffs made in your design decision

None

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by simgunz** [comment]
@michael-genson Thanks for promptly fixing #3359. This may interest you as well given that is quite related.

**Reply by Kuchenpirat** [reply]
Just added a PR that should add the functionality you are looking for: #3376

**Reply by simgunz** [reply]
Great! Thanks.
